test/common/stats/thread_local_store_test.cc

```diff
@@ -80,6 +80,63 @@ class StatsThreadLocalStoreTest : public testing::Test, public RawStatDataAlloca
     store_->addSink(sink_);
   }
 
+  void validateMerge(std::string hist_name, std::vector<uint64_t> h1_cumulative_values,
+                     std::vector<uint64_t> h2_cumulative_values,
```

Comment:
if you are not going to make these member vars (suggested below) you should definitely pass them by const ref.